_posts/en/posts/2016-01-25-segwit-benefits.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: post
+name: segwit-benefits
+id: en-segwit-benefits
+title: Segregated Witness Benefits
+permalink: /en/2016/01/25/segwit-benefits
+version: 1
+---
+1. toc
+{:toc}
```

Comment:
you can also use `{% include _toc.html %}` for TOC style at https://bitcoincore.org/en/faq/optin_rbf/